provider/resource_tasks.go

```diff
@@ -57,7 +57,10 @@ func resourceTasksCreate(d *schema.ResourceData, m interface{}) error {
 		},
 	}
 
-	resp, _ := apiClient.SendRequest("GET", pathVuln, nil, 0)
+	resp, err := apiClient.SendRequest("GET", pathVuln, nil, 0)
+	if err != nil {
+		return err
+	}
 	var jsonData Info
 
 	json.Unmarshal([]byte(resp), &jsonData)
```

Comment:
Maybe check for errors when using Unmarshal

Reply:
Now checking for unmarshall err

Reply:
I don't see that error check?
Maybe something like this? 

```golang
err = json.Unmarshal([]byte(resp), &jsonData)
if err != nil {
  return err
}
```